Example/PinpointKitExample/AppDelegate.swift

```diff
@@ -11,10 +11,10 @@ import PinpointKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
-    var window: UIWindow? = ShakeDetectingWindow(frame: UIScreen.main.bounds)
     
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+    var window: UIWindow?
```

Comment:
Regarding the `var window: UIWindow?` change, specifically? If so, see https://github.com/Lickability/PinpointKit/pull/130/commits/c34451b88746c3662c63b5bfd95c54315da731e5 from https://github.com/Lickability/PinpointKit/pull/130, and this conversation on the PR:

<img width="773" alt="screen shot 2016-09-02 at 4 54 18 pm" src="https://cloud.githubusercontent.com/assets/7883805/18218014/02fa3406-712e-11e6-9072-65276f9ef6cd.png">